doc/Configuration.md

```diff
@@ -398,6 +398,8 @@ license: "<license_key>"
 `kafkaConnects.security.username` : Basic auth username
 
 `kafkaConnects.security.password` : Basic auth password
+ 
+ `jmxScrapePort` : JMX port used to scrap cluster metrics for monitoring. (optional)
```

Comment:
JMX-exporter port

you can also add:
`nodeScrapePort` : node-exporter port used to scrap cluster metrics for monitoring. (optional)